ChemoTracker/components/Search.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { TabNavigator } from 'react-navigation';
+import Navigation from './Navigation.js';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+class Search extends Component {
+  static navigationOptions = {
+    tabBarLabel: "Search",
+    tabBarIcon: () => (<Icon size={24} name="search" color="#EF7A5A" />)
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text>This is the Search page</Text>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
if they (search, profile, etc) all have the same style,  maybe you can put it in a separate place and they all just reference that style. instead of having the same styling code in 3 or more files.  it'll make changing the design across them much easier